### PR TITLE
Connector Health: Fixed SAT for Amazon Ads, Faker, Hubspot, Instagram, Pinterest, Sentry, Zendesk Talk

### PR DESCRIPTION
**Pull request by lazebnyi**
## What
Fixed SAT for Amazon Ads, Faker, Hubspot, Instagram, Pinterest, Sentry, Zendesk Talk

## How
Updated expected records and acceptance-test-config to new format for ignored_fields. Fixed issue with loopback date for pinterest.

### CODE REVIEW

File: airbyte-integrations/connectors/source-sentry/integration_tests/expected_records.jsonl [hunk 1]
```diff
@@ -1,5 +1,6 @@
...o": "bar"}, "unauthorized": false, "url": "http://example.org/foo/bar/"}, "packages": {"my.package": "1.0.0"}, "type": "default", "metadata": {"title": "This is an example Python exception"}, "tags": [{"key": "browser", "value": "Chrome 28.0.1500"}, {"key": "browser.name", "value": "Chrome"}, {"key": "client_os", "value": "Windows 8"}, {"key": "client_os.name", "value": "Windows"}, {"key": "environment", "value": "prod"}, {"key": "level", "value": "error"}, {"key": "sample_event", "value": "yes"}, {"key": "server_name", "value": "web01.example.org"}, {"key": "url", "value": "http://example.com/foo"}, {"key": "user", "value": "id:1", "query": "user.id:\"1\""}], "platform": "python", "dateReceived": "2023-01-17T08:56:35.677376Z", "errors": [], "occurrence": null, "_meta": {"entries": {}, "message": null, "user": null, "contexts": null, "sdk": null, "context": null, "packages": null, "tags": {}}, "crashFile": null, "culprit": "raven.scripts.runner in main", "dateCreated": "2023-01-17T08:55:35Z", "fingerprints": ["3a2b45089d0211943e5a6645fb4cea3f"], "groupingConfig": {"id": "newstyle:2019-10-29", "enhancements": "eJybzDRxY3J-bm5-npWRgaGlroGxrpHxBABcYgcZ"}}, "emitted_at": 1674668829987}
```
Comment (evantahler) [inline airbyte-integrations/connectors/source-sentry/integration_tests/expected_records.jsonl:1]: Without a YAML or code change... what's new with these records?  Did you change the sandbox data?

**Review by evantahler — APPROVED** [review]
This is awesome! 

👍 assuming all the connector tests pass

File: airbyte-integrations/connectors/source-sentry/integration_tests/expected_records.jsonl (hunk 1 above)
Comment (lazebnyi) [reply airbyte-integrations/connectors/source-sentry/integration_tests/expected_records.jsonl:1]: Yes, I was add new test issue event.

**Review by bazarnov — APPROVED** [review]

**Review by grubberr — APPROVED** [review]

**Pull request merged**